vignettes/get_data_inboveg.Rmd

```diff
@@ -26,7 +26,7 @@ The classic relevés can stand alone, be an element of a collection or element o
 Ample selection and export functions toward analysis tools are provided. It also provides standardized lists of species, habitats, life forms, scales etc. 
 Original observations are preserved and a full history of subsequent identifications is saved.
 
-
+More information? Check out https://www.vlaanderen.be/inbo/en-gb/data-applications/inboveg/
```

Comment:
```suggestion
More information? Check out [https://www.vlaanderen.be/inbo/en-gb/data-applications/inboveg/](https://www.vlaanderen.be/inbo/en-gb/data-applications/inboveg/)
```

Like this you can make it a link.  If you wish, you change the text on the link by changing the text inside the `[  ]`.